mitmproxy/proxy/protocol/websocket.py

```diff
@@ -54,7 +52,9 @@ def __init__(self, ctx, handshake_flow):
         extensions = []
         if 'Sec-WebSocket-Extensions' in handshake_flow.response.headers:
             if PerMessageDeflate.name in handshake_flow.response.headers['Sec-WebSocket-Extensions']:
```

Comment:
We'll need to make a few more changes if we remove this `if` 
because there may be case when extensions=[None] which will throw an error in wsproto